crates/typst/src/symbols/symbol.rs

```diff
@@ -46,47 +46,63 @@ pub use typst_macros::symbols;
 #[derive(Debug, Clone, Eq, PartialEq, Hash)]
 pub struct Symbol(Repr);
 
+/// The character of a symbol, possibly with a function.
+#[derive(Debug, Clone, Eq, PartialEq, Hash)]
```

Comment:
This could implement `Copy` and then we can probably get rid of a few `cloned` and the `ref` in the destructuring.